tck/features/MatchAcceptance2.feature

```diff
@@ -0,0 +1,2023 @@
+#
+# Copyright 2016 "Neo Technology",
+# Network Engine for Objects in Lund AB (http://neotechnology.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Feature: MatchAcceptance2
+
+  Scenario: Do not return non-existing nodes
+    Given an empty graph
+    When executing query:
+      """
+      MATCH (n)
+      RETURN n
+      """
+    Then the result should be:
+      | n |
+    And no side effects
+
+  Scenario: Do not return non-existing relationships
```

Comment:
"Do not return non-existing relationships" -> "Do not return non-existent relationships"